### ISSUE TITLE
Roaming Ralph Models and Animations

### ISSUE THREAD

**Issue by blitzcoder** [rdb/blender-egg-importer#27]
Hi, would this add-on be able to import the Ralph model and animation in the samples?

The rigged model (ralph.egg.pz) seems to be imported by parts and not sure how to go about importing the animations

**Comment by rdb** [comment]
Yeah, should work, just be sure to select the base ralph and the animation files at the same time when importing

**Comment by blitzcoder** [comment]
Thanks @rdb importing with animations works, but somehow the eyes and earmuff band are not attached to the head.

Can you provide more tips or info on how to properly import rigged egg models with this add-on?

![image](https://user-images.githubusercontent.com/57479219/217960800-4564ab30-d7ea-446f-b713-b679a49424dc.png)
